lessons/r/dplyrmagrittr/lesson.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: dplyr and magrittr
```

Comment:
Maybe a slightly more informative title, like "Data wrangling with dplyr and magrittr"